aquarius.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<!-- background class added to change backgrounds according to screen size -->
+<html lang="en" class="background">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="stylesheet" href="./assets/style.css" />
+  <link rel="stylesheet" href="./assets/reset.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```

Comment:
You should import libraries such as materialize before you include you own styles

Reset is not one of your libraries, but that definitely needs to come before you import anything else

This makes sure that the css you write can override anything in the framework... so

```

 <link rel="stylesheet" href="./assets/reset.css" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
<link rel="stylesheet" href="./assets/style.css" />
 ```